notebooks/uncertainty_aware_models/labs/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>How these 3 bullet points are correspond to the 5 todos below? It would be clearer to add comments in cells as well I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Done!</p>